File: iSeeCars.Business/Helpers/ConsoleHelper.cs
namespace iSeeCars.Business.Helpers
{
    public static class ConsoleHelper
    {
        public static void Pause()
        {
            Console.WriteLine("Press any key to continue...");
            Console.ReadKey();
            Console.Clear();
        }
        public static void ClearLastInput()
        {
            int line = Console.CursorTop - 1;
            Console.SetCursorPosition(0, line);
            Console.Write(new string(' ', Console.WindowWidth));
            Console.SetCursorPosition(0, line);
        }
        public static void PrintErrorMessage(string message)
        {
            Console.ForegroundColor = ConsoleColor.Red;
            Console.WriteLine(message);
            Console.ForegroundColor = ConsoleColor.Yellow;
        }
    }
}

File: iSeeCars.java/src/main/java/iseecars/helper/ConsoleHelper.java
package iseecars.helper;
import static java.lang.System.out;
import static java.lang.System.in;
public class ConsoleHelper {
    public static void pause(){
    out.println("Press Enter to continue...");
    try{
        in.read();
    } catch (Exception ignored){}
     clearConsole();
    }

    public static void clearConsole(){
        for (int i = 0; i < 50; i++){
            out.println();
        }
    }
    public static void printErrorMessage(String message){
        out.println("\u001B[31m" + message + "\u001B[0m");
    }
}
